script.js

```diff
@@ -0,0 +1,122 @@
+// array to keep the books
+
+let stock = [];
+
+// Get HTML elements
+// const title = document.getElementById('title');
+// const author = document.getElementById('author');
```

Comment:
- [ ] Kindly consider removing all commented code from your base code, this makes your code messy and hard to read
- [ ] [OPTIONAL] Overall make sure to avoid using comments in your code, you can name your functions and variables meaningful names this will act as comments for your code

Reply:
Thank you @Ismailco, we will delete all commented code as per your kind suggestion.